src/main/kotlin/ro/hibyte/betting/service/EventService.kt

```diff
@@ -41,6 +45,27 @@ class EventService(
         eventRepository.save(existingEvent)
     }
 
+    fun checkEventsEndDate() {
+        val events = eventRepository.findAll()
```

Comment:
we only need to check open events

Reply:
Moreover, you can do the timestamp filtering also in the Query, and not fetch all events

Reply:
Create a query method that checks only events with status OPEN